var/spack/repos/builtin/packages/fujitsu-fftw/package.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.pkg.builtin.fftw import FftwBase
+from spack.error import SpackError
+
+
+def target_check(spec):
+    if spec.target != "a64fx":
+        error_msg = ("It can only be built on an A64FX machine.\n")
+        raise SpackError(error_msg)
+
+
+class FujitsuFftw(FftwBase):
+    """FFTW (Fujitsu Optimized version) is a comprehensive collection of
+    fast C routines for computing the Discrete Fourier Transform (DFT)
+    and various special cases thereof.
+
+    It is an open-source implementation of the Fast Fourier transform
+    algorithm. It can compute transforms of real and complex-values
+    arrays of arbitrary size and dimension.
+    Fujitsu Optimized FFTW is the optimized FFTW implementation targeted
+    for A64FX CPUs.
+
+    For single precision build, please use precision value as float.
+    Example : spack install fujitsufftw precision=float
+    """
+
+    _name = 'fujitsufftw'
+    homepage = "https://github.com/fujitsu/fftw3"
+
+    version('master', git='https://github.com/fujitsu/fftw3.git')
```

Comment:
don't you mean `branch=fj_master` here?

Reply:
I fix it to fj_master.

Reply:
I couldn't understand the comment correctly.